presentation/src/main/java/team/jsv/icec/ui/main/mosaic/detect/adapter/DetectedFaceAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package team.jsv.icec.ui.main.mosaic.detect.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import team.jsv.icec.ui.main.mosaic.detect.model.DetectFaceInfoViewItem
+import team.jsv.presentation.databinding.ViewholderDetectedFaceBinding
+
+class DetectedFaceAdapter(private val listener: DetectedFaceListener) :
```

Comment:
전달하는 콜백이 한 개 이상이라면, 인터페이스로 묶어서 보내는게 좋을 수 있지만 하나라면 람다를 사용해 보는건 어떨까요?
```suggestion
class DetectedFaceAdapter(private val onClick: (Int) -> Unit) :
```

Reply:
[37b3a2f](https://github.com/JobSeek-Void/ic-ec/pull/18/commits/37b3a2fc6ab8f594c6200163e8402ce83b9e053a) 리뷰 반영 완료